bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -1084,9 +1084,7 @@ public void run() {
                             // check whether journal file is over file limit
                             if (shouldRolloverJournal) {
                                 // if the journal file is rolled over, the journal file will be closed after last
-                                // entry is force written to disk. the `bc` is not used anymore, so close it to release
-                                // the buffers in `bc`.
-                                IOUtils.close(LOG, bc);
```

Comment:
why is this call removed?

Reply:
As the comment says, the `logFile` object is closed in the syncThread after the last entry is persisted. 

Since the `BufferedChannel` (`bc`) is a component of the `JournalChannel` (`logFile`), in this PR I have associated the `logFile.close()` to also close the internal buffered channel, since `logFile` is the "owner" of the buffered channel.

Reply:
thanks for the explanation.